src/types/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface Post {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+}
+
+export interface User {
+  id: number;
+  name: string,
+  username: string;
+  email: string;
+}
+
+export interface Comment {
+  postId: number,
```

Comment:
Replace the comma with a semicolon to maintain consistency in the interface properties. 🧹